backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Mod.java

```diff
@@ -88,4 +90,16 @@ public void setNewText(final String newText) {
         NodeParser.getNodeFromExpression("./quotedText[2]", this.node).orElseThrow();
     newTextNode.setTextContent(newText);
   }
+
+  /**
+   * Checks weather a quotedText was used for a substitution
+   *
+   * <p>If not it is probably a quotedStructure
```

Comment:
is it ok to not close the `<p>`-tag?

Reply:
Resolved [here](https://github.com/digitalservicebund/ris-norms/commit/541335dde8a7cba0ab3809e30156b04645bb0aaa)